AppMakefile.mk

```diff
@@ -251,12 +251,15 @@ $(BUILDDIR)/$(PACKAGE_NAME).tab: $(foreach platform, $(TOCK_ARCHS), $(BUILDDIR)/
 .PHONY:	all
 all:	$(BUILDDIR)/$(PACKAGE_NAME).tab size
 
-# The size target accumlates dependencies in the platform build rule creation
+# The size target accumulates dependencies in the platform build rule creation
 .PHONY: size
 
 .PHONY: debug
 debug:	$(foreach platform, $(TOCK_ARCHS), $(BUILDDIR)/$(call ARCH_FN,$(platform))/$(call ARCH_FN,$(platform)).userland_debug.lst)
 
+.PHONY: lst
+lst:	$(foreach platform, $(TOCK_ARCHS), $(BUILDDIR)/$(call ARCH_FN,$(platform))/$(call ARCH_FN,$(platform)).lst)
+
```

Comment:
Can you add some comments above this rule and the `make debug` rule to explain which  one you use when?